### DISCUSSION TITLE
Audio doesn't work

### DISCUSSION THREAD

**Discussion by Nojiko75** [pedroSG94/RootEncoder#959] [Q&A] [answered]
Hello @pedroSG94,

I have some issues since you updated the rtmp library in this commit: https://github.com/pedroSG94/rtmp-rtsp-stream-client-java/commit/646cedf32455dfb6ad2b431bba3db5502eeb0283

Indeed, the audio during stream video doesn't work anymore, espacially since you changed the `SrsFlvMuxer` class by the `RtmpClient` class and I don't know why.

For example, this code works:
```kotlin
fun RtmpCamera.prepareAudioRtmp(config: AudioConfig) {
    srsFlvMuxer.setIsStereo(config.isStereo)
    srsFlvMuxer.setSampleRate(config.sampleRate)
}
```

But this code doesn't work:
```kotlin
fun RtmpCamera.prepareAudioRtmp(config: AudioConfig) {
    rtmpClient.setAudioInfo(config.sampleRate, config.isStereo)
}
```
Any ideas ?

Regards

**Comment by pedroSG94** [comment]
Hello, 
It seems that you are using your own implementation of rtmp module. I need know what are you doing exactly to know the problem.
First of all:
- Can you share me a logcat?
- Can you share me a code example of your implementation to know the problem?

**Reply by Nojiko75** [reply]
Hello @pedroSG94,

Unfortunately, I can't show you a logcat or a code example. Indeed, my issue come from my implementation.
So, I tried to fix it and it was a success.
Thank you for your answer.

Regards

**Comment by pedroSG94** [comment, marked as answer]
I'm glad that all is solved.
If you found an error in library and you can share the fix using a PR it will be helpful.

Also, if possible, mark any response as aswer to let know other users that all is solved
